spec/controllers/users_controller_spec.rb

```diff
@@ -19,8 +19,16 @@
   describe "POST create" do
     context "valid input details" do
       
-      before { post :create, user: generate_attributes_for(:user) }
-
+      let(:stripe_helper) { StripeMock.create_test_helper }
+      before do
+        StripeMock.start
```

Comment:
This is OK for now, but the way it should work will be exposed in week 7. Also StripeMock is an excellent choice :).
I will only say that you should know how to mock and stub out service object so you do not hit service. This is doing the same but fairly well wrapped in a library.